tlspuffin/src/tcp/mod.rs

```diff
@@ -794,4 +808,42 @@ mod tests {
 
         info!("{}", shutdown);
     }
+
+    #[test]
+    fn test_wolfssl_buffer_under_read() {
```

Comment:
This test currently fail with:
```
thread 'tcp::tests::test_wolfssl_buffer_under_read' panicked at 'called `Result::unwrap()` on an `Err` value: DeserializeBadVarint', tlspuffin/src/tcp/mod.rs:832:71
```
but the deseralized trace has been produced on the same machine with the same build options :(

Reply:
Commit #204086e seems to fix it, but why ? Anyway